TLM/TLM/Manager/Impl/SpeedLimitManager.cs

```diff
@@ -252,18 +242,18 @@ public class SpeedLimitManager : AbstractGeometryObservingManager, ICustomDataMa
 		/// </summary>
 		/// <param name="customSpeedLimit"></param>
 		/// <returns></returns>
-		public float ToGameSpeedLimit(ushort customSpeedLimit) {
-			if (customSpeedLimit == 0)
-				return MAX_SPEED;
-			return (float)customSpeedLimit / 50f;
+		public float ToGameSpeedLimit(float customSpeedLimit) {
+			return SpeedLimit.IsZero(customSpeedLimit)
+				       ? MAX_SPEED
+				       : customSpeedLimit;
 		}
 
 		/// <summary>
 		/// Converts a lane speed limit to a custom speed limit.
 		/// </summary>
 		/// <param name="laneSpeedLimit"></param>
 		/// <returns></returns>
-		public ushort LaneToCustomSpeedLimit(float laneSpeedLimit, bool roundToSignLimits=true) {
+		public ushort LaneToCustomSpeedLimit_Deprecated(float laneSpeedLimit, bool roundToSignLimits=true) {
```

Comment:
If nothing uses this, why keep it?

Reply:
Happy to remove it.

On oneliners/split lines - the general rule is "if it looks longer than 100 characters (approximately, no hard break), it is going to be split no matter what", if it is below that, I am trying to make it an oneliner. This is consistent once you know the rule :)

Modifying the entire codebase with the same style is a great idea but right now it is a no-no because there's outstanding PR with Harmony which changes a lot of files, and I am trying to keep my changes within some reasonable limits.

Reply:
I don't have a problem with the change overall, but when making this PR you need to consider how the rest of TM:PE is (mostly) coded. We need to stay consistent with the codebase for this PR, and then if you/we want to make these optimizations/formatting changes later, we should do it in another PR. Having half-formatted code with this format and half-formatted code with the other format won't end up well.

Reply:
Removed